src/components/TextareaComposer/styling/SuggestionList.scss

```
@@ -11,4 +12,49 @@
+
+  .str-chat__suggestion-list__item-content {
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    min-width: 0;
+  }
+
+  .str-chat__suggestion-list__item-title {
+    @include fonts.text-caption-default;
+    @include utils.ellipsis-text;
+    text-align: start;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)